### PR TITLE
newsletter: Nov 19

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: _posts/2019-11-24-foreman-community-newsletter---november-2019.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Welcome back to the Foreman Newsletter! After a few months full of conferences 
+that kept our community team very busy, we are now back for good :) 
+
+
+### 1.24 RC2 && Katello 3.14 RC1 are out!
+
+The first release candidate for [1.24][1_24_rc2] was released! That is not a mistake,
+we had to skip RC1. We'd be glad to get feedback and report [any bugs][issues] that you find.
+
```
Comment (tbrisker) [inline _posts/2019-11-24-foreman-community-newsletter---november-2019.md:29]:
```suggestion
Please [take part](https://community.theforeman.org/t/foreman-1-24-test-week/16268) in our testing of the release candidates!
```

File: _posts/2019-11-24-foreman-community-newsletter---november-2019.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+- community
+excerpt: |
+ We're back! CFP still open for Fosdem and CfgMgmtCamp and lots of releases!
+---
+
+Welcome back to the Foreman Newsletter! After a few months full of conferences 
+that kept our community team very busy, we are now back for good :) 
+
+
+### 1.24 RC2 && Katello 3.14 RC1 are out!
```
Comment (tbrisker) [inline _posts/2019-11-24-foreman-community-newsletter---november-2019.md:20]: unless this goes out today we'll be at RC3 already

File: _posts/2019-11-24-foreman-community-newsletter---november-2019.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+* [Open Source Camp](https://community.theforeman.org/t/open-source-camp-on-foreman/15007)
+A full day of everything Foreman, hosted by NETWAYS!
+In case you missed it, read all about it in our
+[blog](_posts/2019-11-07-open-source-camp-on-foreman-2019.md)!
+
+* [Foreman TechDay 2019](https://community.theforeman.org/t/foreman-techday-2019-after-osad-event/14052)
+Right after OSAD, ATIX organized a hackday for Foreman.
+
+* [Norwalk IO](https://community.theforeman.org/t/norwalk-io-managing-your-systems-with-foreman-and-katello/16088)
+John Mitch presented "Managing your systems with Foreman and Katello” at Norwalk IO".
```
Comment (tbrisker) [inline _posts/2019-11-24-foreman-community-newsletter---november-2019.md]:
```suggestion
John Mitch presented "Managing your systems with Foreman and Katello” at Norwalk IO.
```

**Review by tbrisker — COMMENTED** [review]
looks good, a few inline comments

File: _posts/2019-11-24-foreman-community-newsletter---november-2019.md (hunk 2 above)
Comment (orrabin) [reply _posts/2019-11-24-foreman-community-newsletter---november-2019.md:20]: I'd be glad if this goes out today/tomorrow to make sure people see the CFP dates before they end.

**Review by tbrisker — APPROVED** [review]
Thanks @orrabin !

**Pull request merged**